packages/manager/apps/web/client/app/domain/domain.html

```diff
@@ -1,158 +1,156 @@
+                    data-text="{{:: 'domain_tab_DNSSEC' | translate }}"
+                    data-state="{{:: ctrlDomain.getState() + '.dnssec' }}"
+                    data-state-params="ctrlDomain.$stateParams">
+                </oui-header-tabs-item>
+                <oui-header-tabs-item
+                    data-text="{{:: 'domain_tab_TASKS' | translate }}"
+                    data-state="{{:: ctrlDomain.getState() + '.tasks' }}"
+                    data-state-params="ctrlDomain.$stateParams">
+                </oui-header-tabs-item>
+                <oui-header-tabs-item
+                    data-text="{{:: 'hosting_tab_menu_emails' | translate }}"
+                    data-href="{{ '#/configuration/email-domain/' + ctrlDomain.domain.name + '?tab=MAILING_LIST' }}"
+                    data-state-params="ctrlDomain.$stateParams">
+                </oui-header-tabs-item>
+        </oui-header-tabs>
+    </div>
 
-            <div data-ng-if="!ctrlDomain.domain.isExpired">
-                <wuc-autorenew-invite data-ng-if="!ctrlDomain.isAllDom"
-                                      data-product-type="domain"
-                                      data-service-infos="ctrlDomain.domainInfos"
-                                      data-guide-url="{{:: ctrlDomain.autorenewGuide }}"
-                                      data-autorenew-url="{{:: ctrlDomain.autorenewUrl }}">
-                </wuc-autorenew-invite>
-                <div data-ovh-alert="{{alerts.tabs}}"></div>
+    <div class="container-fluid px-5 mt-5">
+        <div class="alert alert-warning" role="alert" data-ng-if="ctrlDomain.domain.isExpired">
+            <div data-ng-bind-html="'common_service_expired' | translate: { t0: '<strong>' + ctrlDomain.domain.displayName + '</strong>' }"></div>
+        </div>
 
-                <div data-ui-view="domainView" ></div>
-            </div>
+        <div data-ng-if="!ctrlDomain.domain.isExpired">
+            <wuc-autorenew-invite data-ng-if="!ctrlDomain.isAllDom"
+                                  data-product-type="domain"
+                                  data-service-infos="ctrlDomain.domainInfos"
+                                  data-guide-url="{{:: ctrlDomain.autorenewGuide }}"
+                                  data-autorenew-url="{{:: ctrlDomain.autorenewUrl }}">
+            </wuc-autorenew-invite>
+            <div data-ovh-alert="{{alerts.tabs}}"></div>
+
+            <div data-ui-view="domainView"></div>
         </div>
+    </div>
 
-        <div id="currentAction" class="modal fade currentAction" tabindex="-1" role="dialog" aria-hidden="true">
-            <div class="modal-dialog">
-                <div id="modal-container" class="modal-content" data-ng-include="ctrlDomain.stepPath"></div>
-                <div id="modal-help" class="help4wizards"></div>
-            </div>
+    <div id="currentAction" class="modal fade currentAction" tabindex="-1" role="dialog" aria-hidden="true">
```

Comment:
⚠️ Caution!

I encourage you to stick with this: 

```suggestion
    <div id="currentAction" class="modal fade currentAction" role="dialog">
```

See: https://github.com/ovh/manager/commit/5cda1fbf136fd7a5d52ed20ac6c900edc3c5df47

Reply:
Ok good to know (this changes shoudn't appear, as I didn't refactor all the domain page). Will apply it